gnomad_qc/v3/annotations/generate_qc_annotations.py

```diff
@@ -185,6 +186,17 @@ def generate_allele_data(ht: hl.Table) -> hl.Table:
 def generate_ac(mt: hl.MatrixTable) -> hl.Table:
     """
     Creates Table with QC samples, QC samples removing children and release samples raw and adj ACs.
+
+    Returned table contains the following annotations:
```

Comment:
Took me a couple reads to figure out that "Returned" was an adjective and not a verb. Can we say instead, "Returns table containing..."?